.github/workflows/trivvy.yaml

```diff
@@ -0,0 +1,33 @@
+---
+name: Trivy vulnerability scanner
```

Comment:
Is it "Trivy" or "Trivvy"? You use both names...

Reply:
yeah. trivy is correct, made it to use that everywhere